MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -1377,6 +1361,15 @@ Table *MantidUI::createDetectorTable(
       }
       const QString isMonitorDisplay = isMonitor ? "yes" : "no";
       colValues << QVariant(specNo) << QVariant(detIds);
+      if (isScanning) {
+        std::set<int> timeIndexSet;
+        for (auto def : spectrumInfo.spectrumDefinition(wsIndex)) {
+          timeIndexSet.insert(int(def.second));
+        }
+
+        QString timeIndexes = createTruncatedList(timeIndexSet);
+        colValues << QVariant(timeIndexes);
+      }
```

Comment:
Yes that is what I meant. I suppose you would see the time indices as [1,2,2,3].